pkg/lib/feature/botprotection/errors.go

```diff
@@ -0,0 +1,11 @@
+package botprotection
+
+import (
+	"github.com/authgear/authgear-server/pkg/api/apierrors"
+)
+
+var Failed = apierrors.Invalid.WithReason("Failed")
+
+var ErrVerificationFailed = Failed.NewWithCause("verification failed", apierrors.StringCause("VerificationFailed"))
+
+var ErrVerificationServiceUnavailable = Failed.NewWithCause("verification service unavailable", apierrors.StringCause("VerificationServiceUnavailable"))
```

Comment:
Do not use cause here. Instead of, define them as follows

```golang
var ErrVerificationFailed = apierrors.Forbidden.WithReason("BotProtectionVerificationFailed").New("bot protection verification failed")

var ErrServiceUnavailable = apierrors.ServiceUnavailable.WithReason("BotProtectionServiceUnavailable").("bot protection service unavailable")
```